cclient/serialization/json/get_balances.c

```diff
@@ -56,6 +56,95 @@ retcode_t json_get_balances_serialize_request(
   return ret;
 }
 
+retcode_t json_get_balances_deserialize_request(const serializer_t *const s,
```

Comment:
let's use `TYPE const * const IDENTIFIER` instead of `const TYPE *const IDENTIFIER` for keeping consistency and readability.

Reply:
I'll add that to `CONTRIBUTING.md` at some point 👍